src/coldHydrogen/coldHydrogen.h

```diff
@@ -47,7 +47,7 @@ auto coldHydrogen( Float tev, int ncold, Float transContinWeight,
    * symmetric in beta
    */
 
-  Float de, x, massMolecule, bp, scatLenC, scatLenI, wt;
+  Float de, x, massMolecule, bp, scatLenC, scatLenI;//, wt;
```

Comment:
What is `Float`?

Reply:
float or double. I think when I first started writing this I was advised to keep float vs. double vague